src/core/CMakeLists.txt

```diff
@@ -72,6 +73,7 @@ list(APPEND DCCORE_PUBLIC_HEADERS
   gestures/PinchGesture.h
   gestures/PinchGestureRecognizer.h
   ImageJpegDecompressor.h
+  ../log.h
```

Comment:
Last thing, can we move log.* to the core folder and get rid of "include_directories(src)" in the top-level CMakeLists?